.github/workflows/run_examples_polynomial.yml

```diff
@@ -29,7 +29,8 @@ jobs:
 
     - name: Start ert-storage
       run: |
-        ert-storage &
+        ert api &
```

Comment:
It is slightly odd that we have to use `ert` to launch storage, continued by running `ert3` commands 🤔 Would it be an idea to expose something along the lines of:

`ert3 service storage start`

as suggested in https://github.com/equinor/ert/issues/1507?

It could start of as a immediate exec of `ert api` 🤷🏻